src/com/lilithsthrone/game/Game.java

```diff
@@ -655,46 +645,75 @@ public static void exportGame(String exportFileName, boolean allowOverwrite) {
 			System.err.println("playerCharacter saving failed!");
 			Main.game.addEvent(new EventLogEntry(Main.game.getMinutesPassed(), "<style='color:"+PresetColour.GENERIC_TERRIBLE.toWebHexString()+";'>Partial Save Fail<b>", "playerCharacter failure"), false);
 		}
-
+		
+		Map<Document, String> saveFiles = new HashMap<>();
+		List<String> keepFiles = Util.newArrayListOfValues("player.xml"); // Always keep the main file
+		saveFiles.put(doc, "player");
+		
 		// Add all NPCs:
 		try {
+			String id = "";
 			for(GameCharacter character : Main.game.getNPCMap().values()) {
+				id = character.getId().replace(",", "_");
+				doc = Main.getDocBuilder().newDocument();
 				Element characterNode = doc.createElement("NPC");
-				game.appendChild(characterNode);
+				doc.appendChild(characterNode);
 				character.saveAsXML(characterNode, doc);
+				saveFiles.put(doc, id);
+				keepFiles.add(id+".xml"); // Add .xml for the sake of removeAll
 			}
 		} catch(Exception ex) {
 			System.err.println("NPC saving failed!");
 			ex.printStackTrace();
 			Main.game.addEvent(new EventLogEntry(Main.game.getMinutesPassed(), "<style='color:"+PresetColour.GENERIC_TERRIBLE.toWebHexString()+";'>Partial Save Fail<b>", "NPC failure"), false);
 		}
-
-
+		
+		// Add all offspringSeed:
+		try {
+			String id = "";
+			for(OffspringSeed offspringSeed : Main.game.getOffspringSeedMap().values()) {
+				id = offspringSeed.getId().replace(",", "_");
+				if(!new File("data/saves/"+exportSaveName+"/"+id+".xml").exists()) {
```

Comment:
I assume offspringSeed won't change after creation so it's save to have an exists() check?

Reply:
Yes